packages/dds/merge-tree/src/referencePositions.ts

```diff
@@ -11,14 +11,18 @@ import { PropertySet, MapLike } from "./properties";
 export const reservedTileLabelsKey = "referenceTileLabels";
 export const reservedRangeLabelsKey = "referenceRangeLabels";
 
-export const refGetTileLabels = (refPos: ReferencePosition): string[] | undefined =>
+export function refTypeIncludesFlag(refPos: ReferencePosition, flags: ReferenceType): boolean {
```

Comment:
Thank you for adding this - I was getting tired of disabling the lint warning every time I wanted to check if a reference was SlideOnRemove